styles/styles.css

```
@@ -717,11 +911,371 @@ body {
+}
+
+.advantages-item-ice-cream::before {
+  background-image: url('../images/ice-cream-mini.svg');
+}
+
+.advantages-item-cow::before {
+  background-image: url('../images/cow.svg');
+}
+
+.advantages-item-thermometre::before {
+  background-image: url('../images/thermometre.svg');
+}
+
+.breadcrumbs-item::after {
+  content: '';
+  width: 12px;
+  height: 12px;
+  margin: auto;
+  background-image: url('../images/double-arrow.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.breadcrumbs-item:last-child::after {
+  content: none;
+}
+
+.button-social {
+  display: block;
+  width: 24px;
+  height: 24px;
+  border: 2px solid #fec7d5;
+  border-radius: 50%;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.button-social-tg {
+  background-image: url('../images/telegram.svg');
+}
+
+.button-social-vk {
+  background-image: url('../images/vk.svg');
+}
+
+.button-social-youtube {
+  background-image: url('../images/youtube.svg');
+}
+
+.button-social:hover {
+  border: 2px solid #fcd5df;
+  border-radius: 50%;
+  background-color: #2d3440;
+}
+
+.button-social:active {
+  border: 2px solid #fdc2d1;
+  border-radius: 50%;
+  background-color: #fcd5df;
+}
+
+.button-social:focus {
+  border: 2px solid #2d3440;
+  border-radius: 50%;
+  background-color: #2d3440;
+}
+
+.advantages-discription {
+  margin: 0;
+}
+
+.product-card-button {
+  width: 32px;
+  height: 32px;
+  margin-left: auto;
+  border: 0;
+  border-radius: 50%;
+  background-color: #ff2f64;
+}
+
+.product-card-button:hover {
+  border: 2px solid #ff2f64;
+  background-color: #fcfcfc;
+}
+
+.product-card-button:hover path {
+  fill: #2d3440;
+}
+
+.product-card-button:active {
+  border: 2px solid #ff2f64;
+  background-color: #ffcbd8;
+}
+
+.product-card-button:active path {
+  fill: #2d3440;
+}
+
+.product-card-button:focus {
+  border: 2px solid #2d3440;
+  background-color: #ff2f64;
+}
+
+.footer-link-heart {
+  display: flex;
+  font-weight: 700;
+  gap: 8px;
+}
+
+.footer-link-heart::before {
+  content: '';
+  display: block;
+  flex-shrink: 0;
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/heart.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.footer-link-product {
+  margin-left: 24px;
+}
+
+
+.other-taste-yellow {
+  position: relative;
+  display: flex;
+  width: 100px;
+  margin: auto 41px auto 37px;
+}
+
+.other-taste-yellow::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100px;
+  height: 100px;
+  margin: auto;
+  border-radius: 50%;
+  background: #ffcbd8;
+}
+
+.other-taste-pink {
+  position: relative;
+  display: flex;
+  width: 100px;
+}
+
+.other-taste-pink::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  width: 100px;
+  height: 100px;
+  margin: auto;
+  border-radius: 50%;
+  background: #ffcbd8;
+}
```

Я бы их стилизовал через общий класс, поскольку не вижу в них отличий вообще (кроме марджина, но его можно доп. классом дать)

```suggestion
.other-taste-yellow {
  position: relative;
  display: flex;
  width: 100px;
}

.other-taste::before {
  content: '';
  position: absolute;
  z-index: -1;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  width: 100px;
  height: 100px;
  margin: auto;
  border-radius: 50%;
  background: #ffcbd8;
}

.other-taste-yellow {
  margin: auto 41px auto 37px;
}
```